editor/editor.tsx

```diff
@@ -800,30 +800,38 @@ function Editor(props: RouteComponentProps) {
                             <br />
                             FIND
                         </span>
-                        <span
-                            title="Use Larger Font"
-                            className="side-panel-button"
-                            onClick={() => {
-                                setIsfontZoomIn(!isFontZoomIn);
-                            }}
-                        >
+                        <span title="Change Font Size" className="side-panel-button">
                             {getIconSVG(ICONS.TEXT, 23, 23)}
-                            +
                             <br />
-                            LARGER
-                        </span>
-                        <span
-                            title="Use Larger Font"
-                            className="side-panel-button"
-                            onClick={() => {
-                                setIsfontZoomOut(!isFontZoomOut);
-                            }}
-                        >
-                            {getIconSVG(ICONS.TEXT, 15, 15)}
-                            -
-                            <br />
-                            SMALLER
+                            FONT
```

Comment:
Perhaps, users might think that they can change font families by seeing `FONT`. How about `FONT SIZE` if this fit into the narrow side panel.